src/mixins/enum-mixin.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, Vue } from 'vue-property-decorator'
+import {
+  GetCorpFullDescription,
+  GetCorpInfoObject,
+  GetCorpNumberedDescription
+} from '@bcrs-shared-components/corp-type-module'
+import { AssociationTypes } from '@/enums'
+
+/**
+ * Mixin that provides some useful enum utilities.
+ */
+@Component({})
+export default class EnumMixin extends Vue {
+  // from external module
+  getCorpTypeInfo = GetCorpInfoObject
+  getCorpTypeDescription = GetCorpFullDescription
+  getCorpTypeNumberedDescription = GetCorpNumberedDescription
```

Comment:
Looks like these aren't used. Please clean up.

Reply:
@seeker25 Please don't resolve my conversations. I use open conversations as a bookmark of code to re-review -- otherwise I have to re-review everything.

Reply:
Ok, sorry about that